ownCloudScreenshotsTests/SnapshotHelper.swift

```diff
@@ -0,0 +1,294 @@
+//
```

Comment:
What is the license for this file? Are you sure, we can use it?

Reply:
It is part of the Fastlane project that use MIT License:
https://github.com/fastlane/fastlane/blob/master/LICENSE